src/controller/manualAttendance.js

```diff
@@ -0,0 +1,21 @@
+import db from '../db';
+
+module.exports = {
+
+    manualUpdatedAttendance: (req, res, next) => {
+        db.manual.manualUpdate(req, res).then((data) => { // gets all data manually entered by employees which is not reviewed
+            res.json(data)
+        }).catch((err) => {
```

Comment:
write this in short code everywhere

.catch((err) => {
 +            res.json(err);
 +        })

to

.catch(err =>res.json(err))

everywhere such small expression should be written in short code